src/components/TodoModal/TodoModal.tsx

```diff
@@ -1,42 +1,73 @@
-import React from 'react';
+import React, { useEffect, useState } from 'react';
 import { Loader } from '../Loader';
+import { Todo } from '../../types/Todo';
+import { getUser } from '../../api';
+import { User } from '../../types/User';
+import { SelectedTodo } from '../../types/SelectedTodo';
+
+type Props = {
+  todo: Todo;
+  setSelectedTodo: (todo: SelectedTodo) => void;
+};
+
+export const TodoModal: React.FC<Props> = ({
+  todo,
+  setSelectedTodo,
+}: Props) => {
+  const [user, setUser] = useState<User | null>(null);
+  const { id, title, completed, userId } = todo;
+  const [isLoading, setIsLoading] = useState(true);
```

Comment:
here too start with false
```suggestion
  const [isLoading, setIsLoading] = useState(false);
```